### PR TITLE
doc: documentation about io_flush --dependent option

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
@komainu8 implementation is not added yet, but in advance, draft review to get feedback is good manner.  could you do draft review?

File: doc/source/reference/commands/io_flush.rst [hunk 1]
```diff
@@ -310,6 +315,61 @@ and all columns:
...
+
+- An related index column (There is source column in TABLE_NAME)::
+
+    io_flush --target_name TABLE_NAME_OF_INDEX_COLUMN.INDEX_COLUMN
+
+The following ``io_flush`` flushes all changes in database, all tables
+and all columns:
+
+.. groonga-command
+.. include:: ../../example/reference/commands/io_flush/dependent_yes.log
```
Comment (komainu8) [inline doc/source/reference/commands/io_flush.rst]: I can't find this file.
Do you have not made this file yet?

Comment (kenhys) [reply doc/source/reference/commands/io_flush.rst]: Sorry, I've fogot it.

**Comment by kenhys** [comment]
needless PR, so close.

**Pull request closed without merging**